src/permission-set.js

```diff
@@ -717,26 +715,27 @@ class PermissionSet {
 
   /**
    * @method save
-   * @param [aclUrl] {String} Optional URL to save the .ACL resource to. Defaults
-   *   to its pre-set `aclUrl`, if not explicitly passed in.
+   * @param [options={}] {Object} Options hashmap
+   * @param [options.aclUrl] {String} Optional URL to save the .ACL resource to.
+   *   Defaults to its pre-set `aclUrl`, if not explicitly passed in.
+   * @param [options.contentType] {string} Optional content type to serialize as
    * @throws {Error} Rejects with an error if it doesn't know where to save, or
    *   with any XHR errors that crop up.
-   * @return {Promise<Request>}
+   * @return {Promise<SolidResponse>}
    */
-  save (aclUrl, rdf, webClient) {
-    aclUrl = aclUrl || this.aclUrl
+  save (options = {}) {
```

Comment:
A neat trick is to squash the next couple of lines into the function signature like this, which I also find easier to read:

```js
save ({ aclUrl = this.aclUrl, contentType = DEFAULT_CONTENT_TYPE }) { ... }
```